.github/workflows/CI.yml

```diff
@@ -600,7 +602,7 @@ jobs:
 
           # Now we can create the makefile
           ./bootstrap.sh
-          ./configure CC=${CC} CFLAGS="${CFLAGS}" --prefix=${LOCAL} ${EXTRA_OPTIONS}
+          ./configure CC=${CC} CFLAGS="${CFLAGS}" --prefix=${LOCAL} ${EXTRA_OPTIONS} --with-gmp-lib=$(find ~/.julia/ -type d -wholename "*/lib/julia")
```

Comment:
What is this meant to do?

Reply:
We talked about it, this is to find the GMP used by Julia, so we link against the same one. That makes sense! I would suggest to add a comment to explain that

But the above code won't work in general. This will work: put the following into a `.jl` file somewhere:
```julia
using GMP_jll, Artifacts
function gmp_artifact_dir()
    artifacts_toml = joinpath(dirname(dirname(Base.pathof(GMP_jll))), "StdlibArtifacts.toml")

    # If this file exists, it's a stdlib JLL and we must download the artifact ourselves
    if isfile(artifacts_toml)
        meta = artifact_meta("GMP", artifacts_toml)
        hash = Base.SHA1(meta["git-tree-sha1"])
        if !artifact_exists(hash)
            dl_info = first(meta["download"])
            download_artifact(hash, dl_info["url"], dl_info["sha256"])
        end
        return artifact_path(hash)
    end

    # Otherwise, we can just use the artifact directory given to us by GMP_jll
    return GMP_jll.find_artifact_dir()
end

```
Then you can do this:
```
julia -e 'include("path/to/bla.jl"); print(gmp_artifact_dir())
```
This will print the prefix path you need.

Reply:
Thanks!!